CMakeLists.txt

```diff
@@ -108,12 +114,26 @@ endif()
 # ------------------
 # OpenCL interoperability
 option(SYCL_CTS_ENABLE_OPENCL_INTEROP_TESTS
-       "Enable OpenCL interoperability tests." ON)
+       "Enable OpenCL interoperability tests." OFF)
```

Comment:
I suggest changing OpenCL interop test enabling option default in a separate patch. I don't think we should change it in the PR, which adds CUDA backend tests.

Reply:
Done